pkg/gomod/file/parser_test.go

```diff
@@ -4,6 +4,8 @@ import (
 	"os"
 	"testing"
 
+	"github.com/spf13/afero"
```

Comment:
can you group these together to be more compact?

Reply:
@michalpristas  it's the vs code go-extension. it uses goimports and it's separating stdlib packages from non-stdlib packages

Reply:
Yes that's how I imagined it grouped by std/github

Reply:
Ups... now i see - the empty lines... I thought you don't want the separation between std and non std pkgs. should be fixed now.

Reply:
👍 no problem @marpio, thank you very much, I would address it in separate PR otherwise.
DO you plan any other commits here?

Reply:
@michalpristas no, I hope that's all. Did you find anything else?

Reply:
nono, i think it looks good.
I would like to have at least one more eye on this though